ci/build-package-set.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+shopt -s nullglob
+
+psroot=$(dirname "$(dirname "$(realpath "$0")")")
+
+if [[ "${CI:-}" ]]; then
+  if [[ "$(echo $psroot/CHANGELOG.d/breaking_*)" ]]; then
+    echo "Skipping package-set build due to unreleased breaking changes"
+    exit 0
+  fi
+
+  # Go to where we actually built PureScript in a previous step.
+  cd "$psroot/sdist-test"
+
+  # !!! `--local-doc-root`? Yes. Our ci/build.sh script, for some reason,
+  # doesn't put the purs binary in a location included in the PATH that
+  # `stack exec` uses, so `stack exec which purs` doesn't work. I don't have a
+  # good explanation for why that is, or why local-doc-root is the path that
+  # *does* contain the binary, but... well, here we are and I'm sorry.
+  pursdir=$(dirname "$(stack path --local-doc-root)")/bin
```

Comment:
Oh, right... I actually fixed an issue in #4139 using this same solution. #4139 fixed #4138 because we added the `--haddock` flag in #4172 as a way to fix #4073.

Also, I'm pretty sure #4172 didn't fix #4073. There's no docs for [PureScript on Hoogle](https://hackage.haskell.org/package/purescript).

Reply:
The docs stopped being published in v0.14.0, which I believe is the same release where we split `purescript-cst` and `purescript-ast` into their own libraries.

Reply:
Ah... I thought it looked familiar! Thanks.

Figuring out what's up with the missing Hoogle docs is more than I want to tackle right now, but at least I can update that comment to be a little more helpful.